.devcontainer/Dockerfile.console

```diff
@@ -9,6 +9,7 @@ ENV OC_PLUGIN_NAME=$OC_PLUGIN_NAME
 USER root
 CMD eval "oc login $OC_URL -u $OC_USER -p $OC_PASS --insecure-skip-tls-verify" && \
     /opt/bridge/bin/bridge -public-dir=/opt/bridge/static \
+    -i18n-namespaces plugin__$OC_PLUGIN_NAME \
```

Comment:
Hello @lavocatt , could you please explain the purpose of adding `i18n-namespaces` here? is it added for translation purpose.

Reply:
That is from the console plugin template. I've basically imported all the configuration from https://github.com/openshift/console-plugin-template/commit/2b4dbe4b4313e258be1754815614cf7d47f45c11

Reply:
Got it. Thanks @lavocatt